app/resources/query_form.html.template

```diff
@@ -126,7 +126,7 @@
           {% endif %}
           <input id="query_name" name="query_name"
               class="long-text-input"
-              value="{{params.query_name}}" placeholder="example: Will Smith">
+              value="{{params.query_name}}" placeholder="{% trans 'Will Smith, Will, Smith' %}">
```

Comment:
Maybe better to say "Will Smith, Will **or** Smith" to clarify that the user should put just one of them.